### PR TITLE
prey: add livecheck, update caveats

### LINKED ISSUE #99252: Update prey instructions
#### General troubleshooting steps

- [x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [ ] I have retried my command with `--force`.
- [ ] I ran `brew update-reset && brew update` and retried my command.
- [ ] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [x] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [ ] I made doubly sure this is not a [checksum does not match](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md) error.

#### Description of issue

`prey` cask has the following instructions:

>    Prey requires your API key, found in the bottom-left corner of
>    the Prey web account Settings page, to complete installation.
>    The API key may be set as an environment variable as follows:

Source: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/prey.rb

These instructions seem to be outdated.

Source: https://help.preyproject.com/article/316-unattended-install-for-macos

>  You can find your Setup API Key on your About page, on the Setup API Key section. Click on it to save to the clipboard.

Maybe add the direct link to the instructions as well: https://panel.preyproject.com/settings/about

### PR DESCRIPTION
**Pull request by miccal**
Closes https://github.com/Homebrew/homebrew-cask/issues/99252.

### CODE REVIEW

**Comment by miccal** [comment]
Failure is expected:
```
Error: Failure while executing; `/usr/bin/sudo -E -- /usr/bin/env LOGNAME=runner USER=runner USERNAME=runner /usr/sbin/installer -pkg /usr/local/Caskroom/prey/1.9.9/prey-mac-1.9.9-x64.pkg -target /` exited with 1. Here's the output:
54
No API Key provided. Stopping here.
```
All other changes check out.

File: Casks/prey.rb [hunk 1]
```diff
@@ -3,20 +3,28 @@
...
+
   pkg "prey-mac-#{version}-x64.pkg"
 
   uninstall pkgutil:   "com.prey.agent",
             launchctl: "com.prey.agent"
 
   caveats <<~EOS
-    Prey requires your API key, found in the bottom-left corner of
-    the Prey web account Settings page, to complete installation.
+    Installing Prey requires your Setup API Key, found on your
```
Comment (vitorgalvao) [inline Casks/prey.rb:21]: Do do you mean “found on the”?

File: Casks/prey.rb [hunk 2]
```diff
@@ -3,20 +3,28 @@
...
+    About page on the Setup API Key section, as explained here:
+
+      https://help.preyproject.com/article/316-unattended-install-for-macos
+
     The API key may be set as an environment variable as follows:
 
-      HOMEBREW_NO_ENV_FILTERING=1 API_KEY="abcdef123456" brew install --cask prey
+      HOMEBREW_NO_ENV_FILTERING=1 API_KEY="foobar123" brew install --cask prey
```
Comment (vitorgalvao) [inline Casks/prey.rb:28]: Let’s make it more explicit: `write_key_here`.

File: Casks/prey.rb (hunk 1 above)
Comment (miccal) [reply Casks/prey.rb:21]: I thought it best to follow the wording given in the link as closely as possible.

File: Casks/prey.rb (hunk 2 above)
Comment (miccal) [reply Casks/prey.rb:28]: I thought it best to follow the example given in the link.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**